packages/api-docs-builder/ApiBuilders/ComponentApiBuilder.ts

```diff
@@ -195,11 +201,72 @@ async function annotateComponentDefinition(api: ReactApi) {
         end = start;
       }
     },
+
+    ExportNamedDeclaration(babelPath) {
+      if (!api.filename.includes('mui-base')) {
+        return;
+      }
+
+      let node: babel.Node = babelPath.node;
+
+      if (node.declaration == null) {
```

Comment:
Is this going to work with the `Unstable_NumberInput`?

Reply:
It shouldn't matter as the component is exported as `NumberInput` from its source file. The "unstable_" prefix is added in its index.ts, which this script doesn't care about.